src/blackd/__init__.py

```diff
@@ -32,6 +32,7 @@
 PYTHON_VARIANT_HEADER = "X-Python-Variant"
 SKIP_STRING_NORMALIZATION_HEADER = "X-Skip-String-Normalization"
 SKIP_MAGIC_TRAILING_COMMA = "X-Skip-Magic-Trailing-Comma"
+PREVIEW = "X-PREVIEW"
```

Comment:
```suggestion
PREVIEW = "X-Preview"
```
I know HTTP headers are case-insensitive, but we might as make it consistent with the other constants :)